src/stream.rs

```diff
@@ -885,6 +875,17 @@ impl SendBuf {
         Ok(())
     }
 
+    /// Returns the highest contiguously ACKed offset.
+    pub fn ack_off(&self) -> u64 {
+        match self.acked.iter().next() {
+            // Only consider the initial range if it contiguously covers the
+            // start of the stream (i.e. from offset 0).
+            Some(std::ops::Range { start: 0, end }) => end,
+
+            Some(_) | None => 0,
```

Comment:
`||`?

Reply:
Nope, `|` is the actual operator to match multiple patterns in Rust https://doc.rust-lang.org/1.2.0/book/patterns.html#multiple-patterns